src/client/actions/elems.js

```diff
@@ -0,0 +1,35 @@
+export const ADD_ELEM = 'ADD_ELEM';

```

Comment:
good variable names are important for others when they are reading: what is elem ? use todo instead.